### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-06-30

### DISCUSSION THREAD

**Discussion by fire** [omigroup/gltf-extensions#92] [General]
This meeting will be on 6/30/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- [gltf extensions](https://github.com/omigroup/gltf-extensions) Discuss support table
- [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) Feedback
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- End of the meeting show and tell period

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
# Notes:
- Feedback on VRM 1.0 Beta Spec
  - Might be useful to add additional phoneme blendshapes. Most notably "ん" or "nn" whis is present in MMD https://www.deviantart.com/xoriu/art/MMD-Facial-Expressions-Chart-341504917
  - The changelog shows VRMC_materials_hdr_emissiveMultiplier as a supported extension, but it should likely be swapped ojut for KHR_materials_emissive_strength https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_emissive_strength/README.md
- KHR_audio
  - @antpb  to add audio.schema.json and fix glTF root schema
- OMI_collider
  - Initial working implementation in Third Room and UnityGLTF export
  - gltf-transform plugin needed for that workflow
  - Hulls can probably be auto generated in all engines, they probably shouldn't include the hull data in the glTF file

**Comment by madjin** [comment]
**VRM discussion** some feedback from [avaer ](https://github.com/avaer) who wrote the best VRM implementation on the web: https://github.com/webaverse/app

Summarized:

> There's a few places where this spec is written poorly (as in could not be implemented unless you copied existing code). Not really clear what the data types are here https://github.com/vrm-c/vrm-specification/blob/master/specification/VRMC_vrm-1.0-beta/lookAt.md. The spec leaves out details, like it seems to assume you are using unity with some person's existing code since it doesn't fully tell you the data formats. For this to be a good spec there needs to be all of the JSON fields, their types, and allowed values, clearly specified
not just "in the unity component it works like this".
